addons/mrp/report/mrp_report_bom_structure.py

```diff
@@ -96,7 +96,7 @@ def _get_bom(self, bom_id=False, product_id=False, line_qty=False, line_id=False
         bom_quantity = line_qty
         if line_id:
```

Comment:
I would simply use
```python
if line_id and line_qty:
```
to avoid unnecessary operations

Reply:
Thanks @Whenrow for the review, however `bom_quantity` is used in several other places:

- https://github.com/odoo/odoo/blob/12.0/addons/mrp/report/mrp_report_bom_structure.py#L113
- https://github.com/odoo/odoo/blob/12.0/addons/mrp/report/mrp_report_bom_structure.py#L121
- https://github.com/odoo/odoo/blob/12.0/addons/mrp/report/mrp_report_bom_structure.py#L138

So if line_qty is null, bom_quantity will be too, and so it will crash and we will have to add conditions in several places!

Reply:
Sorry I thought it was `== 0`. In that case we can leave it as you did